zuul.d/jobs.yaml

```diff
@@ -2189,3 +2189,172 @@
       - OS:ubuntu-xenial
       - Arch:arm64
       - BuildType:Compile
+
+# Temp Kubernetes cloud-provider-openstack jobs on citynetwork cloud, these
+# jobs will be removed later.
+- job:
+    name: cloud-provider-openstack-test-citynetwork
+    parent: golang-test
+    description: |
+      Base job for all types of cloud-provider-openstack test jobs
+    pre-run: playbooks/cloud-provider-openstack-test/pre.yaml
+    post-run: playbooks/cloud-provider-openstack-test/post.yaml
+    nodeset: ubuntu-xenial-citynetwork
```

Comment:
Adding `nodeset` into abstract and non-running job looks not a good practice, because there is no `nodeset` define in child running job, so that cause developer is not easy to know which nodeset will be used, have to check parent job again, so I think we should add `nodeset` into running child directly, thought?

Reply:
good idea. We can add them it to the offcial jobs in the next PR once the temp ones are well tested.